docker-compose-ut.yml

```diff
@@ -11,14 +11,26 @@ services:
       POSTGRES_DB: sampledb
       POSTGRES_USER: pguser
       POSTGRES_PASSWORD: topsecret
+    tty: true
+    healthcheck:
+      test: nc -z localhost 5432 || exit -1
+      interval: 5s
+      timeout: 5s
+      retries: 3
+      start_period: 5s
 
   api:
     image: acme/api:1.0
+    build: api
```

Comment:
Couldn't get tests to run without this. @gnschenker I probably should have reread your blog post but just incase this is interesting.

Reply:
yeah, the DB must be ready *before* the tests run :-)